mlir/test/xmir/e2e/resnet50_torch_part_0.mlir

```diff
@@ -0,0 +1,20 @@
+// RUN: rocmlir-driver -host-pipeline partition,highlevel -targets %arch %s | rocmlir-gen -ph -print-results -rand 1 -rand_type float -fut resnet50_part_0 --verifier clone - | rocmlir-driver -host-pipeline xmodel -kernel-pipeline full | xmir-runner --shared-libs=%linalg_test_lib_dir/libmlir_rocm_runtime%shlibext,%conv_validation_wrapper_library_dir/libconv-validation-wrappers%shlibext,%linalg_test_lib_dir/libmlir_runner_utils%shlibext,%linalg_test_lib_dir/libmlir_c_runner_utils%shlibext,%linalg_test_lib_dir/libmlir_async_runtime%shlibext --entry-point-result=void | FileCheck %s
```

Comment:
Nit: Can we use `xmodel,runner` and `mlir-cpu-runner -O2` here instead? It'll simplify debugging.

Reply:
I prefer to keep the XMIR flow which follows the same steps. Debugging xmir-runner is generally equivalent to rocmlir-driver.

Reply:
I prefer not to, as xmir-runner will both run a bunch of CPU-side passes we don't need and there's the confusion of having non-trivial compilation in "runner"